src/features/pages/partners/deriv-prime/hero/index.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { StaticImage } from 'gatsby-plugin-image'
+import { hero_wrapper, container, hero_img, hero_contact_button } from './styles.module.scss'
+import { Localize, localize } from 'components/localization'
+import Typography from 'features/components/atoms/typography'
+import Flex from 'features/components/atoms/flex-box'
+import dclsx from 'features/utils/dclsx'
+import Button from 'features/components/atoms/button'
+
+const Hero = () => {
+    return (
+        <Flex.Box
+            bgcolor="black"
+            container="fixed"
+            as="section"
+            align="center"
+            className={hero_wrapper}
+        >
+            <div className={hero_img}>
+                <StaticImage
+                    src="../../../../../images/common/affiliate/partners-banner.png"
+                    alt={localize(
+                        '_t_A professional trader searching for a liquidity solutions provider_t_',
+                    )}
+                    layout="fullWidth"
+                    loading="eager"
+                    placeholder="blurred"
+                    formats={['avif', 'webp', 'auto']}
+                    style={{ height: '100%' }}
+                    objectPosition="center"
+                    className={dclsx('visible-larger-than-tablet')}
+                />
+                <StaticImage
+                    src="../../../../../images/common/deriv-prime/hero-mobile.png"
+                    alt={localize(
+                        '_t_A professional trader searching for a liquidity solutions provider_t_',
+                    )}
+                    layout="fullWidth"
+                    loading="eager"
+                    placeholder="blurred"
+                    formats={['avif', 'webp', 'auto']}
+                    style={{ height: '100%' }}
+                    className={dclsx('visible-phone-and-tablet')}
+                />
+            </div>
+            <Flex.Box
+                direction="col"
+                align="center"
+                padding_inline="5x"
+                className={container}
+                md={{ padding_inline: '0x' }}
+            >
+                <Typography.Heading align="center" size="xlarge" textcolor="white">
+                    <Localize translate_text="_t_Global liquidity for all_t_" />
+                </Typography.Heading>
+                <Typography.Paragraph align="center" textcolor="white" size="large" pb="12x">
+                    <Localize translate_text="_t_Deep, reliable liquidity with fast execution for professionals, seasoned institutions, start-ups, and more_t_" />
+                </Typography.Paragraph>
+
+                <Button.Primary
+                    textsize="large"
+                    hero
+                    id="dm-nav-deriv-prime-contact"
+                    onClick={() => {
```

Comment:
can we use `<a>` here ? 🤔

Reply:
We are using the button primary here

Reply:
@aswathy-deriv Yup you are correct, But I remember while reviewing @hasan-deriv PR [here ](https://github.com/binary-com/deriv-com/pull/5433/files#diff-531a38740c1811b8ae406b4c3a8db62cd8b12fe5ed29d9a9565a4c3c6c29b24fR59-R63), he used the `LinkButton.Primary` which is the `<a>`, anything we faced that we changed to button primary? I'm just trying to learn here and see which one to use in the future 🙏